lib/beaker-puppet/version.rb

```diff
@@ -1,3 +1,3 @@
 module BeakerPuppet
-  VERSION = '1.22.1'
+  VERSION = '1.23.0'
```

Comment:
I would consider this much more a bugfix than a new feature since and bump the version to 1.22.2 (maybe the merged PR title could be adjusted to `Fix install_puppet_on on el8`)?

Reply:
done